Bottom: 4ae672abe0bd947cd1214507dfb391e5771b84a1
Top:    ba2fa45a2ce66e29c539d87bde21d32455ae6bc5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-05 17:29:50 +0000

Refresh of board-outline-polygon-generati

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 82098ca..14f125c 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1615,7 +1615,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
     DrawLayerGroup (drawn_groups [i], drawn_area);
 
 #if 1
-    if (!global_view_2d &&
+    if (!global_view_2d && i > 0 &&
         drawn_groups[i] >= min_phys_group &&
         drawn_groups[i] <= max_phys_group &&
         drawn_groups[i - 1] >= min_phys_group &&
